lib/ringbuf/build.rs

```diff
@@ -0,0 +1,8 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    build_util::expose_cpu_info();
```

Comment:
Curious why this needed a `expose_cpu_info()`?

Reply:
`expose_cpu_info()` provides a cfg flag on what type of atomics to use.

Reply:
Aah I see. Thanks!